File: app/src/main/java/com/codingblocks/restapiretrofitjson/MainActivity.java
package com.codingblocks.restapiretrofitjson;

import android.content.Intent;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.Button;

import com.codingblocks.restapiretrofitjson.adapters.UserAdapter;

/**
 * Created by Harshi on 7/5/2017.
 */

public class MainActivity extends AppCompatActivity {

    public static final String TAG = "USERS";

    RecyclerView rvUserList;
    UserAdapter userAdapter;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button btnOpenUsers,btnOpenAlbums,btnOpenTodos,btnOpenPosts;

        btnOpenAlbums = (Button) findViewById(R.id.btnImages);
        btnOpenUsers = (Button) findViewById(R.id.btnUsers);
        btnOpenTodos = (Button) findViewById(R.id.btnTodos);
        btnOpenPosts = (Button) findViewById(R.id.btnPosts);

        btnOpenUsers.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent i = new Intent(MainActivity.this,UsersActivity.class);
                startActivity(i);
            }
        });

        btnOpenTodos.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent i = new Intent(MainActivity.this,TodosActiviy.class);
                startActivity(i);
            }
        });

        btnOpenAlbums.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent i = new Intent(MainActivity.this,AlbumsActivity.class);
                startActivity(i);
            }
        });

        btnOpenPosts.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent i = new Intent(MainActivity.this,PostsActivity.class);
                startActivity(i);
            }
        });

    }
}

File: app/src/main/java/com/codingblocks/restapiretrofitjson/MainActivity.kt
package com.codingblocks.restapiretrofitjson

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import com.codingblocks.restapiretrofitjson.activities.AlbumActivity
import com.codingblocks.restapiretrofitjson.activities.PostsActivity
import com.codingblocks.restapiretrofitjson.activities.TodosActivity
import com.codingblocks.restapiretrofitjson.activities.UsersActivity
import com.squareup.picasso.Picasso

import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        Picasso.with(this@MainActivity).load("https://cdn0.iconfinder.com/data/icons/large-glossy-icons/512/Large_group.png").into(ivUsers)
        Picasso.with(this@MainActivity).load("http://icons.iconarchive.com/icons/paomedia/small-n-flat/1024/post-it-icon.png").into(ivPosts)
        Picasso.with(this@MainActivity).load("https://droidveda.com/ij/wp-content/uploads/2013/09/album-maker-icon.png").into(ivAlbums)
        Picasso.with(this@MainActivity).load("https://xinote.com/application/web/views/templates/xiview/images/home/todolist_icon.png").into(ivTodos)

        ivUsers.setOnClickListener({
            startActivity(Intent(this@MainActivity, UsersActivity::class.java))
        })

        ivPosts.setOnClickListener({
            startActivity(Intent(this@MainActivity, PostsActivity::class.java))
        })

        ivAlbums.setOnClickListener({
            startActivity(Intent(this@MainActivity, AlbumActivity::class.java))
        })

        ivTodos.setOnClickListener({
            startActivity(Intent(this@MainActivity, TodosActivity::class.java))
        })


    }
}
